examples/temp-sensors/flow.yaml

```diff
@@ -1,92 +1,145 @@
-collections:
-  # Information about a temperature sensor, which may produce many temperature readings
-  temperature/sensors:
-    schema: schemas.yaml#/$defs/tempSensor
-    key: [/id]
+endpoints:
```

Comment:
We should use the `examples/examples.db` endpoint instead of adding this.

Reply:
I updated all of this -- let me know if it looks good to merge.